src/MOCK_DB_DATA/library-items/layout-templates/layout-template-expandable-first-col.ts

```diff
@@ -0,0 +1,64 @@
+import { LibraryItemIds } from '../library-item-ids';
+import { LibraryItem, LibraryItemType } from 'src/app/views/shared/models/library-item.model';
+import { LibraryItemConstants as CONST } from '../library-item-constants';
+
+export const LayoutTemplateExpandableFirstCol: LibraryItem[] = [{
+  id: LibraryItemIds.LayoutTemplateExpandableFirstCol,
+  title: 'Layout med ekspanderbart innhold i første kolonne',
+  type: LibraryItemType.fhiAngular,
+  exampleHtml: getExampleHtml(),
+  codeHtml: getCodeHtml(),
+  documentationHtml: getDocumentationHtml(),
+  fullScreenEnabled: true
+}];
+
+
+/*
+ * Return value is ignored if LibraryItemType is not html
+ */
+function getExampleHtml(): string {
+  return ``;
+}
+
+/*
+ * Return empty string to use a copy of exampleHtml as codeHtml.
+ * Return null to remove Code from library-item.
+ */
+function getCodeHtml(): string | null {
+  return `
+<div class="container">
+	<div class="fhi-expandable-first-col">
+		<div class="row">
+			<div class="col-12 col-md-auto">
```

Comment:
Hmm... maybe `<div class="col-12 col-md-auto fhi-expandable-first-col__drawer-container">` after all... not sure...

Reply:
I'd say no, because we don't need any specific css attributes attached